pennylane/ops/functions/equal.py

```diff
@@ -99,3 +114,14 @@ def equal(
                 return False
 
     return getattr(op1, "inverse", False) == getattr(op2, "inverse", False)
+
+
+def equal_measurements(op1, op2):
```

Comment:
Is this a good idea? If `qml.equal` would extend in the future to other functions, i thought that breaking up `qml.equal` to smaller functions would make things easier to read.

Reply:
Yes! This is definitely easier to read